onnxruntime/core/common/threadpool.cc

```diff
@@ -143,23 +160,28 @@ class ThreadPool::Impl : public TaskThreadPool {
 #endif
   }
 
-  void ParallelForRange(int64_t first, int64_t last, std::function<void(int64_t, int64_t)> fn) {
+  void ParallelFor(int64_t total, int64_t unit_size, std::function<void(int64_t, int64_t)> fn) {
 #ifdef USE_OPENMP
 #pragma omp parallel for
-    for (int64_t id = first; id < last; ++id) {
-      fn(id, id + 1);
+    for (int64_t first = 0; first < total; first += unit_size) {
+      int64_t last = (first + unit_size < total) ? first + unit_size : total;
+      fn(first, last);
     }
 #else
-    Barrier barrier(static_cast<unsigned int>(last - first));
+    int64_t remainder = total - unit_size * NumThreads();
+    int64_t spawnedThreads = (unit_size == 1 ? total : NumThreads()) - 1;
```

Comment:
spawnedThreads -- naming convention. should be spawned_threads.

Reply:
Fixed.